src/js/share/utils.js

```diff
@@ -224,7 +224,19 @@ export const checkNextBlock = (array) => {
     if(array[i].childBlocks_.length > 0) {
       for(let j = 0; j < array[i].childBlocks_.length; j++) {
         if(array[i].style.colourPrimary === array[i].childBlocks_[j].style.colourPrimary) {
-          array[i].childBlocks_[j].style.colourPrimary = array[i].childBlocks_[j].style.colourTertiary;
+          const colourPrimaryTemp = array[i].childBlocks_[j].style.colourPrimary;
```

Comment:
```suggestion
          const colourPrimaryTemp = array[i].childBlocks_[j].style.colourPrimary;
```

PLEASE CLARIFY: Does Temp refer to temperature or to temporary here? It is honestly not clear to me. 
SUGGESTION: In general, please do not use ambiguous abbreviations, or abbreviations at all if they can easily be avoided.

Reply:
![Screenshot from 2020-04-29 17-34-34](https://user-images.githubusercontent.com/48412313/80615790-6c6f6080-8a40-11ea-96e6-bde997cbbdd0.png)

Does this look better now?